src/Stratis.Bitcoin.Tests/FinalizedBlockInfoRepositoryTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Tests.Common;
+using Xunit;
+
+namespace Stratis.Bitcoin.Tests
+{
+    public class FinalizedBlockInfoRepositoryTest : TestBase
+    {
+        private readonly ILoggerFactory loggerFactory;
+        private readonly Network network;
+
+        public FinalizedBlockInfoRepositoryTest() : base(KnownNetworks.StratisRegTest)
+        {
+            this.loggerFactory = new LoggerFactory();
+            this.network = KnownNetworks.StratisRegTest;
```

Comment:
Ah actually you don't need to do this as it is set in the base, so you can just use `this.Network` everywhere ;)

Reply:
fixed